src/olympia/devhub/utils.py

```diff
@@ -183,6 +184,51 @@ def _merged_messages():
     }
 
 
+def find_previous_version(addon, file, version):
+    """
+    Find the most recent previous version of this add-on, prior to
+    `version`, that can be used to compare validation results or
+    issue upgrade warnings.
+    """
+    if not addon:
+        return
+
+    version = Version(version)
```

Comment:
lolwut
could less confusing if the first `version` was called `version_string` (assuming that's what it's supposed to be)